dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/engine/command/impl/AbstractSubmitToArchiveCommand.java

```diff
@@ -43,6 +43,11 @@ public DatasetVersion execute(CommandContext ctxt) throws CommandException {
             }
         }
 
+        String setting = ctxt.settings().getValueForKey(SettingsServiceBean.Key.FQDN);
```

Comment:
Why were those lines needed here? Previously there was nothing about fqdn and siteurl here.

Reply:
Rather than passing siteUrl and new parameter I wanted to use > requestedSettings < We can discuss this approach.